pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -40,10 +40,6 @@
         <Exclude>Compile</Exclude>
       </Dependency>
     </ItemGroup>
-
-    <PropertyGroup>
-    <VersionSuffix>-rc3-$(CLIBuildVersion)</VersionSuffix>
```

Comment:
The version number gets passed via https://github.com/dotnet/core-setup/blob/master/pkg/pack.cmd#L68 as BuildNumberMajor and BuildNumberMinor. With those we end up building a version that looks like `-rc3-004315-00`.